src/UI/Art/SpriteArtist/CommanderOverlayArtist.java

```diff
@@ -128,14 +129,14 @@ public static BufferedImage buildCoPowerBar(Commander co, int[] abilityPoints, d
     for( int i = 0; i < abilityPoints.length; ++i )
     {
       int requiredPower = abilityPoints[i];
-      double diff = requiredPower - currentPower;
+      double diff = (requiredPower - currentPower)/Commander.CHARGERATIO_FUNDS;
```

Comment:
Adding all of the random `CHARGERATIO_FUNDS` everywhere does not help with intuitiveness.

Reply:
I want to get my intent done, which is not exactly the same as what I did, haha.